crates/biome_js_parser/tests/js_test_suite/error/ts_export_declare.ts.snap

```diff
@@ -178,7 +196,24 @@ ts_export_declare.ts:2:16 parse ━━━━━━━━━━━━━━━━
   > 2 │ export declare @decorator abstract class D {}
       │                ^^^^^^^^^^
     3 │ 
+    4 │ declare@
   
   i Decorators are only valid on class declarations, class expressions, and class methods.
   
+ts_export_declare.ts:4:9 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Expected an expression but instead found the end of the file.
+  
+    2 │ export declare @decorator abstract class D {}
+    3 │ 
+  > 4 │ declare@
+      │         
+  
+  i Expected an expression here.
+  
+    2 │ export declare @decorator abstract class D {}
+    3 │ 
+  > 4 │ declare@
+      │         
```

Comment:
Looking at the snapshot, it doesn't seem to contain the new errors. I believe there's something wrong

Reply:
i update this~